copybookreader-plugins/src/test/java/co/cask/hydrator/plugin/CopybookTest.java

```diff
@@ -139,13 +130,18 @@ public void testCopybookReaderWithOutputSchema() throws Exception {
     DataSetManager<Table> outputManager = getDataset(outputDatasetName);
     List<StructuredRecord> output = MockSink.readOutput(outputManager);
     StructuredRecord record = output.get(0);
-    Assert.assertEquals("Expected schema", schema, record.getSchema());
+
     Assert.assertEquals("Expected records", 5, output.size());
-    Assert.assertEquals("Expected schema", null, record.get("DATE"));
+    Assert.assertEquals("Output schema", "{\"type\":\"record\",\"name\":\"record\"," +
```

Comment:
these asserts are guaranteed to fail. Please make sure your tests pass before sending pull requests.